_security/access-control/field-masking.md

```diff
@@ -78,6 +78,16 @@ See [Create role]({{site.url}}{{site.baseurl}}/security/access-control/api/#crea
 
 By default, the Security plugin uses the BLAKE2b algorithm, but you can use any hashing algorithm that your JVM provides. This list typically includes MD5, SHA-1, SHA-384, and SHA-512.
 
+You can override the default algorithm in `opensearch.yml`:
+
+```yml
+plugins.security.masked_fields.algorithm.default: SHA-256
+```
+
+Property | Description
```

Comment:
Do we plan on adding additional settings?

Reply:
> Do we plan on adding additional settings?

There are no plans that I am aware of.

Reply:
Let's remove the setting from the table then: 

```suggestion

```